packages/components/src/theme/ComposerTheme/Components/Button.ts

```diff
@@ -0,0 +1,91 @@
+import { ComponentStyleConfig } from '@chakra-ui/theme';
+
+export const Button: ComponentStyleConfig = {
+  baseStyle: {
+    fontWeight: 'medium',
+    borderRadius: 'md',
```

Comment:
please use the values from our theme https://github.com/cmpsr/composer/blob/with-chakra/packages/components/src/theme/ComposerTheme/Radius/radius.ts

Reply:
changed to use Composer theme radius values.